websites/Z/ZeroTwo/dist/metadata.json

```diff
@@ -38,12 +39,6 @@
 			"title": "Display Timestamp",
 			"icon": "fas fa-stopwatch",
 			"value": true
-		},
-		{
-			"id": "buttons",
-			"title": "Show Buttons",
-			"icon": "fas fa-compress-arrows-alt",
-			"value": false
 		}
```

Comment:
Instead of removing functionality, you could add buttons?

Reply:
last time it was said that buttons that link to some page were prohibited..?

Reply:
You are allowed, with the following exceptions 
![image](https://user-images.githubusercontent.com/64903135/162071543-c681f5ac-d9b5-45b9-ad5d-0488d1929e50.png)

Reply:
to what page am i supposed to link it to, then?